`pylint` gives gives warnings

I checked out the `HEAD` (15e8c693) of `main` and ran `pylint` as suggested in the `README`:
```
$ pylint --rcfile=scripts/.pylintrc scripts
************* Module make
scripts/make.py:607:4: C0116: Missing function or method docstring (missing-function-docstring)
************* Module repos
scripts/repos.py:162:0: C0330: Wrong hanging indentation (remove 2 spaces).
              repo, branch_str, status['SHA1'], status['Dirty']))
            | ^ (bad-continuation)
scripts/repos.py:26:0: E0401: Unable to import 'git' (import-error)
************* Module build
scripts/build.py:60:47: C0326: Exactly one space required after comma
    parser.add_argument('--build-dir', type=str,  metavar='PATH',
                                               ^ (bad-whitespace)
scripts/build.py:146:33: C0326: Exactly one space required after comma
    parser.add_argument('actions',  nargs=argparse.REMAINDER,
                                 ^ (bad-whitespace)
************* Module config
scripts/config.py:162:42: C0326: Exactly one space required after comma
        self.name = '{}_{}_{}'.format(arch,  float_abi.value, name_suffix)
                                          ^ (bad-whitespace)
scripts/config.py:227:0: C0330: Wrong continued indentation (remove 3 spaces).
                                  action != Action.TEST)
                               |  ^ (bad-continuation)
scripts/config.py:226:34: R1714: Consider merging these comparisons with "in" to 'action not in (Action.ALL, Action.TEST)' (consider-using-in)
------------------------------------------------------------------
Your code has been rated at 9.89/10 (previous run: 9.89/10, +0.00)
```

My version of `pylint` is as follows:

```
$ pylint --version
pylint 2.4.2
astroid 2.3.1
Python 3.8.5 (default, Jan 27 2021, 15:41:15)
[GCC 9.3.0]
```

--- a/scripts/build.py
+++ b/scripts/build.py
@@ -57,7 +57,7 @@ def parse_args_to_config() -> Config:
     parser.add_argument('--source-dir', type=str, metavar='PATH', default=cwd,
                         help='location of the LLVM Embedded '
                              'Toolchain for Arm source checkout (default: .)')
-    parser.add_argument('--build-dir', type=str,  metavar='PATH',
+    parser.add_argument('--build-dir', type=str, metavar='PATH',
                         help='directory to use for build '
                              '(default: ./build-<revision>)')
     parser.add_argument('--install-dir', type=str, metavar='PATH',
@@ -143,7 +143,7 @@ def parse_args_to_config() -> Config:
                         help='number of parallel threads to use in Make/Ninja '
                         '(default: number of CPUs, {})'.format(cpu_count),
                         default=cpu_count)
-    parser.add_argument('actions',  nargs=argparse.REMAINDER,
+    parser.add_argument('actions', nargs=argparse.REMAINDER,
                         choices=util.values_of_enum(Action),
                         help='actions to perform, a list of:\n'
                              '  prepare - check out and patch sources\n'

--- a/scripts/config.py
+++ b/scripts/config.py
@@ -159,7 +159,7 @@ class LibrarySpec:
         self.float_abi = float_abi
         self.arch_options = arch_options
         self.other_flags = other_flags
-        self.name = '{}_{}_{}'.format(arch,  float_abi.value, name_suffix)
+        self.name = '{}_{}_{}'.format(arch, float_abi.value, name_suffix)
 
     @property
     def target(self):
@@ -223,8 +223,7 @@ class Config:  # pylint: disable=too-many-instance-attributes
 
         if not args.actions or Action.ALL.value in args.actions:
             self.actions = set(action for action in Action
-                               if action != Action.ALL and
-                               action != Action.TEST)
+                               if action not in (Action.ALL, Action.TEST))
             if Action.TEST.value in args.actions:
                 self.actions.add(Action.TEST)
         else:

--- a/scripts/make.py
+++ b/scripts/make.py
@@ -577,10 +577,10 @@ class ToolchainBuild:
                 expected_output_path = os.path.join(smoketest_path,
                                                     output_type)
                 try:
-                    expected_output = open(expected_output_path)
-                    expected_output_str = [
-                        s.rstrip('\n') for s in expected_output.readlines()
-                    ]
+                    with open(expected_output_path) as expected_output:
+                        expected_output_str = [
+                            s.rstrip('\n') for s in expected_output.readlines()
+                        ]
                 except OSError as ex:
                     logging.error("Failed to open %s", expected_output_path)
                     raise util.ToolchainBuildError from ex
@@ -607,6 +607,7 @@ class ToolchainBuild:
         return all_tests_succeeded
 
     def run_tests(self, lib_spec: config.LibrarySpec) -> None:
+        """Runs smoke tests."""
         result = self._run_smoke_tests(lib_spec)
         if not result:
             raise util.ToolchainBuildError

--- a/scripts/repos.py
+++ b/scripts/repos.py
@@ -159,7 +159,7 @@ def print_repositories_status(checkout_path: str) -> int:
         else:
             branch_str = 'Detached'
         print(' - {}: {}, Revision: {}, Dirty: {}'.format(
-              repo, branch_str, status['SHA1'], status['Dirty']))
+            repo, branch_str, status['SHA1'], status['Dirty']))
 
     return 0
 

